src/components/Movie/Movie.js

```diff
@@ -37,4 +48,21 @@ function Movie({ id, posterPath, title, rating , onMovieClick, updateMovieId, up
 	)
 }
 
-export default Movie;
\ No newline at end of file
+export default Movie;
+
+function isURL(props, propName, componentName) {
```

Comment:
Checks if the URL for an image(specifically the poster_path) is a valid image to use.